storybook/preview-body.html

```diff
@@ -0,0 +1,5 @@
+<style>
```

Comment:
Isn't this something we should be able to put into the theme file? 🤔

Reply:
So apparently to affect the styles within the preview, you have to add them here. The theme file is only for everything outside the preview window -- cause that makes _total_ sense!

Reply:
Oh Storybook, you make perfect sense until you don't 😆

Reply:
That said I think the main reason they don't make it intuitive to do this is to make most Storybook out in the wild look the same #stickyBranding